docs/MO_DG/prepare_model/convert_model/Converting_Model.md

```diff
@@ -108,7 +108,8 @@ Framework-agnostic parameters:
   --log_level {CRITICAL,ERROR,WARN,WARNING,INFO,DEBUG,NOTSET}
                         Logger level
   --input INPUT         Quoted list of comma-separated input nodes names with shapes, 
-                        data types, and values for freezing. The shape and value are 
+                        data types, and values for freezing. The order of inputs in converted 
+                        model is the same as order of specified operation names.The shape and value are 
```

Comment:
white-space is missed here: "...names.The..."

Reply:
Fixed.